velox/exec/ExchangeClient.h

```diff
@@ -116,7 +116,7 @@ class ExchangeClient {
   std::shared_ptr<ExchangeQueue> queue_;
   std::unordered_set<std::string> taskIds_;
   std::vector<std::shared_ptr<ExchangeSource>> sources_;
-  bool closed_{false};
+  std::atomic_bool closed_{false};
```

Comment:
Are you making it atomic to ensure safe access from multiple threads?  If so, would you update PR title and description to make this clear.

Reply:
thank you for reivew this pr.
The original meaning of this PR is ensure that the ExchangeClient is closed. we not to push back ExchangeSource for producingSources_ to again request.

Reply:
thansk, roll back to the modification of closed_.